client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/api/UserFilesClient.kt

```diff
@@ -35,4 +49,9 @@ internal class UserFilesClient(private val template: RestTemplate) : FilesOperat
     override fun deleteFile(fileName: String, relativePath: String) {
         template.delete("$USER_FILES_URL${normalize(relativePath)}?fileName=$fileName")
     }
+
+    private fun saveFile(response: ClientHttpResponse, fileName: String): File {
+        val targetPath = Files.createTempFile("biostudies-$fileName", ".tmp")
+        copy(response.body, targetPath, REPLACE_EXISTING); return targetPath.toFile()
```

Comment:
Same here. Actually, this code looks really similar, should it be in a utility or extension method?